conference.js

```diff
@@ -157,13 +166,13 @@ function createInitialLocalTracksAndConnect(roomName) {
     return Promise.all([ tryCreateLocalTracks, connect(roomName) ])
         .then(([tracks, con]) => {
             APP.store.dispatch(mediaPermissionPromptVisibilityChanged(false));
-            if (audioAndVideoError) {
+            if (audioAndVideoError || noVideo) {
```

Comment:
to get to handling 'audioOnlyError' when noVideo option is used

I was trying to simplify this, but I don't have any ideas...